lib/daru/io/exporters/sql.rb

```diff
@@ -26,11 +25,16 @@ class SQL < Base
         #
         #   Daru::IO::Exporters::SQL.new(df, dbh, table).call
         def initialize(dataframe, dbh, table)
-          super(binding)
+          super(dataframe)
+          @dbh       = dbh
+          @table     = table
         end
 
         def call
-          require 'dbi'
+          optional_gem 'dbd-sqlite3', requires: 'dbd/SQLite3'
+          optional_gem 'dbi'
+          optional_gem 'sqlite3'
```

Comment:
Probably better to do in `initialize`, before any other initialization/option checking.